enclave-runner/src/usercalls/mod.rs

```diff
@@ -598,6 +613,13 @@ impl UsercallExtension for UsercallExtensionDefault{
     ) -> IoResult<Option<Box<SyncStream>>> {
         Ok(None)
     }
+
+    fn bind_stream(&self,
+                  _addr: &str,
+                  _local_addr: Option<&mut String>
+                  ) -> IoResult<Option<Box<SyncListener>>> {
+        Ok(None)
+    }
```

Comment:
Don't need this `impl` since it's defaulted. Actually, you can also remove the `impl` for `fn connect_stream`.